### ISSUE TITLE
Consider how user can use this?

### ISSUE THREAD

**Issue by dorafmon** [BuckleTypes/bs-containers#11]
`immutable.re` seems to support `npm` packaging so you only need to do `npm install immutable.re` then you can use it directly in your reason/bs project. Should we also try to have this? @glennsl

**Comment by glennsl** [comment]
Absolutely, but we should wait until we can provide some stability guarantee.

**Comment by bordoley** [comment]
fwiw the immutable.re npm setup at this point is mostly for developer simplicity (eg. me). I'd like to support shipping via npm at least for bucklescript use cases but haven't had time to focus on that. For the time being, opam has been the official release channel.
